commands/configure/index.js

```diff
@@ -95,14 +113,15 @@ const setSkippedExperience = (api, application) => {
 program
   .description('Configures and associates a directory on disk to represent one of your Losant applications and its resources.')
   .action(async (command) => {
-    const userConfig = await loadUserConfig() || {};
-    if (!userConfig.apiToken) {
-      return logError('Must run losant login before running losant configure.');
-    }
+    let userConfig = await loadUserConfig() || {};
     await Promise.all(DIRECTORIES_TO_GENERATE.map((dir) => { return ensureDir(dir); }));
     await Promise.all(LOCAL_META_FILES.map((type) => { return saveLocalMeta(type, {}); }));
-    const api = await getApi({ apiToken: userConfig.apiToken });
-    const getApplication = getApplicationFunc(api);
+    const url = await getApiURL(userConfig);
+    if (!url) { return; }
```

Comment:
is this actually possible?

Reply:
There are two redundant checks that Erin wanted in this file: this one and logError in getApiUrl. I don't know of cases where they could be reached, but Erin would know better than I do.